src/main/java/com/pinkulu/heightlimitmod/forge/ForgeMod.java

```diff
@@ -1,8 +1,14 @@
 package com.pinkulu.heightlimitmod.forge;
 
+import co.uk.isxander.evergreenhud.addon.AddonManager;
 import com.pinkulu.heightlimitmod.HeightLimitMod;
 import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.event.FMLInitializationEvent;
 
 @Mod(modid = HeightLimitMod.MOD_ID, version = HeightLimitMod.MOD_VERSION, name = HeightLimitMod.MOD_NAME, dependencies = "required-after:evergreenhud", clientSideOnly = true)
```

Comment:
the dependencies needs to be `required-before:evergreenhud` otherwise it will register the addon after the loading stage. did you even test this ??

Reply:
![image](https://user-images.githubusercontent.com/45589059/120884307-a2502280-c60c-11eb-978f-3124cebf417b.png)
????????????

Reply:
I mean now it is supposed to be `required-before` now. It was a change to increase performance

Reply:
POINT STILL STANDS /j